en/frames_rover/ackermann_rover.md

```diff
@@ -0,0 +1,246 @@
+# Ackermann Rover
+
+<Badge type="tip" text="main (PX4 v1.16+)" /> <Badge type="warning" text="Experimental" />
+
+An _Ackermann Rover_ controls its direction by pointing the front wheels in the direction of travel — the [ackermann steering geometry](https://en.wikipedia.org/wiki/Ackermann_steering_geometry) compensates for the fact that wheels on the inside and outside of the turn move at different rates.
+This kind of steering is used on most commercial vehicles, including cars, trucks etc.
+
+![Traxxas Stampede VXL](../../assets/airframes/rover/traxxas_stampede_vxl/stampede.jpg)
+
+::: info
+The module does not require the ackermann geometry and will work with any front-steering rover.
+:::
+
+## Airframe Setup
+
+This section will go through the most basic setup that is necessary to start using the rover.
+
+1. To enable the module flash the PX4 rover build onto your flight controller.
+
+   ::: info
+   You can also enable the module in normal builds by adding the following line to your [board configuration](../hardware/porting_guide_config.md) (e.g. for fmu-v6x you might add this line to [`main/boards/px4/fmu-v6x/default.px4board`](https://github.com/PX4/PX4-Autopilot/blob/main/boards/px4/fmu-v6x/default.px4board)):
+
+   ```sh
+   CONFIG_MODULES_ROVER_ACKERMANN=y
+   ```
+
+   Note that adding the rover module may lead to flash overflow.
+   You may need to disable modules that you do not plan to use, such as those related to multicopter, fixed wing, or the old rover module.
+   :::
+
+2. In the [Airframe](../config/airframe.md) configuration select the _Generic ackermann rover_:
+
+   ![QGC screenshot showing selection of the airframe 'Generic ackermann rover'](../../assets/config/airframe/airframe_ackermann_rover.png)
+
+   Select the **Apply and Restart** button.
+
+   ::: info
+   If this airframe does not show up in the UI, it can alternatively be selected by setting the [SYS_AUTOSTART](../advanced_config/parameter_reference.md#SYS_AUTOSTART) parameter to `50010`.
```

Comment:
We could do this, but normally we explain the QGC minimum build you need to get the ability to install the appropriate airframe - do you know what that is (I may run out of time before I try out latest daily).